x/emissions/module/rewards/rewards.go

```diff
@@ -344,7 +339,7 @@ func GenerateRewardsDistributionByTopicParticipant(
 	if args.TopicReward == nil {
 		return nil, alloraMath.Dec{}, types.ErrInvalidReward
 	}
-	args.Ctx.Logger().Debug(fmt.Sprintf("Generating rewards distribution for topic: %d, block: %d, topicReward: %s", args.TopicId, args.BlockHeight, args.TopicReward.String()))
+	args.Ctx.Logger().Debug("Generating rewards distribution for topic", "topicId", args.TopicId, "block", args.BlockHeight, "topicReward", args.TopicReward.String())
```

Comment:
```suggestion
	args.Ctx.Logger().Debug("Generating rewards distribution for topic", "topicId", args.TopicId, "block", args.BlockHeight, "topicReward", args.TopicReward)
```

To be tested, but as we implement the `MarshalJSON` func in `Dec` I think it should work

Reply:
`TopicReward` is a *alloraMath.Dec`. I believe we do this in other places as well... Signing off.